### DISCUSSION TITLE
Licensing for NAPS2.Sdk

### DISCUSSION THREAD

**Discussion by JohnDoe2193** [cyanfish/naps2#130] [General]
I would like to understand the licensing for NAPS2.Sdk (GPL V2+).

We would like to use NAPS2.Sdk for our internal line of business application. We do not intend to make any changes to SDK or sell our product to anybody else.

Do we still have to publish our proprietary source code so that it is publicly available?

**Comment by cyanfish** [comment]
As noted in the [readme](https://github.com/cyanfish/naps2#license), NAPS2.Sdk and its dependencies are available under the LGPL 2.1 license. This means if you make no changes to the SDK itself you have no obligations to publish your code.

**Reply by JohnDoe2193** [reply]
Thanks!
